app/src/main/java/com/bebetterprogrammer/tictactoe/activities/VsJarvis.kt

```diff
@@ -0,0 +1,221 @@
+package com.bebetterprogrammer.tictactoe.activities
+
+import android.app.AlertDialog
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.appcompat.app.AppCompatActivity
+import com.bebetterprogrammer.tictactoe.BuildConfig
+import com.bebetterprogrammer.tictactoe.R
+import kotlin.random.Random
+import kotlin.random.nextInt
+import kotlinx.android.synthetic.main.activity_gameplay.appBottomLine
+import kotlinx.android.synthetic.main.activity_gameplay.quit
+import kotlinx.android.synthetic.main.activity_vs_jarvis.*
+import kotlinx.android.synthetic.main.result_dialog.view.*
+
+class VsJarvis : AppCompatActivity() {
+    var turn: Int = 0
```

Comment:
VsJarvis class is not needed as we can use the existing class `GamePlayActivity.kt` with Jarvis variant. Just pass the variant via intent and take the decision based on that variant.